src/coreFeatures/User.java

```diff
@@ -0,0 +1,57 @@
+package coreFeatures;
+
+import java.math.BigDecimal;
+
+public class User {
+    private String[] categories = new String[] {"Food", "Snacks and self regret purchases", "Gas", "Entertainment", "Housing", "Subscriptions"};
+
+    private BigDecimal[] spendingByCategory = new BigDecimal[categories.length];
+    
+    private Account[] accounts;
+
+    //set the category size and then make a checkings and savings account
+    public User(int categorySize){
+        categorySetSize(categorySize);
+
+        accounts = new Account[2];
+        accounts[0] = new Account("Checking", BigDecimal.ZERO, false);
+        accounts[1] = new Account("Savings", BigDecimal.ZERO, true);
+    }
+
+    public BigDecimal findNetWorth(){
+        BigDecimal total = BigDecimal.ZERO;
+        for(Account a : accounts){
+            total.add(a.getBalance());
+        }
+        return total;
+    }
+
+    public User(){
+        //25 default value
+        this(25);
+    }
+
+    public Account[] getAccounts(){
+        return accounts;
+    }
+
+    public String[] getCategories(){
+        return categories;
+    }
+
+    public void categorySetSize(int num){
```

Comment:
Is this necessary? Can add, remove, and set category functions handle all this achieves?